flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1781,6 +1781,12 @@ private void setClusterEntrypointInfoToConfig(final ApplicationReport report) {
 
         flinkConfiguration.set(
                 YarnConfigOptions.APPLICATION_ID, ConverterUtils.toString(clusterId));
+
+        // set cluster-id to app id if not specified
```

Comment:
maybe rename the `clusterId` variable to `appId`

Reply:
Jup, good idea.